Set some normal hiliting for eruby tags, same as php opening tags are.

--- ./eruby.vim~	2005-05-12 22:10:03.000000000 +0300
+++ ./eruby.vim	2005-05-12 22:28:43.000000000 +0300
@@ -32,7 +32,7 @@
 syn region erubyComment start=+<%#+ end=#%># keepend
 syn match erubyEnd #\<end\>#
 
-hi link erubyRubyDelim todo
+hi link erubyRubyDelim Delimiter
 hi link erubyComment comment
 hi link erubyEnd rubyControl
 
